data-table/table.js

```diff
@@ -157,6 +157,44 @@ function addColumns(numberOfColumns, index) {
      */
 }
 
+/**
+ * This function enables the user to enter the name of a column. Creates a field for text input
+ * as well as a button that sends input text to addSingleColumn().
+ * @param {object} config   - Table configuration object
+ * @returns {undefined}     - Doesn't return anything
+ */
+function createColumnInputAndBtn(config) {
+    let entryBoxDiv = document.getElementById(config.tableIds.entryBoxDivId);
+    // Creates the field for text input
+    let input = document.createElement("INPUT");
+    input.type = 'text';
+    input.placeholder = "Enter " + config.datumConfig.names;
+
+    // If the user hits enter while in the text box, click the addColumnBtn
+    input.addEventListener("keyup", function(event) {
+        event.preventDefault();
+
+        if (event.code === "Enter") {
+            addColumnBtn.click();
+        }
+    })
+    entryBoxDiv.appendChild(input);
+
+    // Creates the button that will take the user input and send it to addSingleColumn() when clicked
+    let addColumnBtn = document.createElement("button");
```

Comment:
This button is gonna need to get updated to accept a number of columns from the user then pass that number to addMultipleColumns (which I'm realizing should probably be createMultiple columns but that's a separate issue).

Functionally the code looks fine but if you could add a TODO to flag that we'll need to update this code that would be great.